funcx_endpoint/funcx_endpoint/endpoint/interchange.py

```diff
@@ -395,7 +395,8 @@ def process_pending_results():
                         continue
 
                     except Exception as exc:
-                        log.debug(f"Invalid message received: {result}")
+                        if "result" in vars():
+                            log.debug(f"Invalid message received: {result}")
```

Comment:
I think the `{result}` was a well-intentioned idea for debugging, but unfortunately not helpful here: it will quite likely be either non-existent (on first result attempt, per the bug report!), or will be from a previous loop iteration.  Suggest removing the debug line entirely, as without the failed messages result, it adds no value.

Reply:
Agreed. I'll remove it.